webapp/components/admin_console/email_settings.jsx

```diff
@@ -105,6 +111,48 @@ export default class EmailSettings extends AdminSettings {
             );
         }
 
+        let emailNotificationContentsTypeDropdown = null;
+        let emailNotificationContentsHelpText = null;
+        if (window.mm_license.EmailNotificationContents === 'true') {
+            const emailNotificationContentsTypes = [];
+            emailNotificationContentsTypes.push({value: EMAIL_NOTIFICATION_CONTENTS_FULL, text: Utils.localizeMessage('admin.email.notification.contents.full', 'Send full message contents')});
+            emailNotificationContentsTypes.push({value: EMAIL_NOTIFICATION_CONTENTS_GENERIC, text: Utils.localizeMessage('admin.email.notification.contents.generic', 'Send generic description with only sender name')});
+
+            if (this.state.emailNotificationContentsType === EMAIL_NOTIFICATION_CONTENTS_FULL) {
+                emailNotificationContentsHelpText = (
+                    <FormattedHTMLMessage
+                        key='admin.email.notification.contents.full.description'
+                        id='admin.email.notification.contents.full.description'
+                        defaultMessage='Sender name and channel are included in email notifications.</br>Typically used for compliance reasons if Mattermost contains confidential information and policy dictates it cannot be stored in email.'
+                    />
+                );
+            } else if (this.state.emailNotificationContentsType === EMAIL_NOTIFICATION_CONTENTS_GENERIC) {
+                emailNotificationContentsHelpText = (
+                    <FormattedHTMLMessage
+                        key='admin.email.notification.contents.generic.description'
+                        id='admin.email.notification.contents.generic.description'
+                        defaultMessage='Only the name of the person who sent the message, with no information about channel name or message contents are included in email notifications.</br>Typically used for compliance reasons if Mattermost contains confidential information and policy dictates it cannot be stored in email.'
+                    />
+                );
+            }
+
+            emailNotificationContentsTypeDropdown = (
+                <DropdownSetting
+                    id='emailNotificationContentsType'
+                    values={emailNotificationContentsTypes}
+                    label={
+                        <FormattedMessage
+                            id='admin.email.notification.contents.title'
+                            defaultMessage='Email Notification Contents: '
+                        />
+                    }
+                    value={this.state.emailNotificationContentsType}
+                    onChange={this.handleChange}
+                    helpText={emailNotificationContentsHelpText}
```

Comment:
When the value of the dropdown changes, the help text updates to show information about the selected option